File: ex02/ex02_01/ex02_01.js
import promptSync from 'prompt-sync';
const prompt = promptSync();

function main() {
  let name = null;
  let luckyNumber = null;

  name = prompt("Enter your name: ");
  luckyNumber = Number(prompt("Enter your lucky number: "));
  console.log(name + ": Your lucky number is " + String(luckyNumber));
  return 0;
}
main();

File: ex02/ex02_01/ex02_01.c
#include <stdio.h>

int main() {
  char name[20];  
  int lucky_number;

  printf("Enter your name: ");
  scanf("%s", name);
  printf("Enter your lucky number: ");
  scanf("%d", &lucky_number);
  printf("%s: Your lucky number is %d\n", name, lucky_number);
  return 0;
}